src/lpython/python_evaluator.cpp

```diff
@@ -68,7 +200,7 @@ Result<std::unique_ptr<LLVMModule>> PythonCompiler::get_llvm3(
     Result<std::unique_ptr<LCompilers::LLVMModule>> res
         = asr_to_llvm(asr, diagnostics,
             e->get_context(), al, lpm, compiler_options,
-            run_fn, infile);
+            "", infile); // ??? What about run function
```

Comment:
The argument `""` is the `run_fn`. The `run_fn` is not used here. In LFortran all global statements are compiled into 1 function, but in LPython it is compiled into two different functions one for global initialization and the other to execute global statements.

Reply:
Now that we have merged `global_init` and `global_stmts`, we can assign `run_fn` to `global_stmts` and use it.

Then that change would be part of this PR. You can have a look at it in successive commits.